Library/Homebrew/utils/github.rb

```diff
@@ -457,6 +457,18 @@ def workflow_dispatch_event(user, repo, workflow, ref, **inputs)
                   scopes:         CREATE_ISSUE_FORK_OR_PR_SCOPES)
   end
 
+  def create_or_update_release(user, repo, tag, id: nil, name: "", draft: false)
+    url = "#{API_URL}/repos/#{user}/#{repo}/releases"
+    url += "/#{id}" if id
+    method = id ? :PATCH : :POST
+    data = {
+      tag_name: tag,
+      name:     name,
```

Comment:
Hmm, what happens here if you pass an empty string for `name`? Should it default to `tag`?

Reply:
For draft PRs the release title is literally "Draft", so I guess I'm gonna default to tag.